docs/RichTextInput.md

```diff
@@ -171,6 +171,49 @@ const MyRichTextInput = ({ size, ...props }) => (
 );
 ```
 
+## Reference the editor
+
+You might need access to the editor to use [the dedicated API](https://tiptap.dev/docs/editor/api/editor). To do it, you can assign a `ref` in the `onCreate` function in the `editorOptions` prop of your `<RichTextInput>` component which returns the editor:
+
+```tsx
+import { Edit, SaveButton, SimpleForm, TextInput, Toolbar } from 'react-admin';
+import { DefaultEditorOptions, RichTextInput } from 'ra-input-rich-text';
+import { Buuton } from 'ra-ui-materialui';
+import { Editor } from '@tiptap/react';
+
+export const PostEdit = () => {
+    const editorRef = React.useRef<Editor>(null);
+
+    return (
+        <Edit>
+            <SimpleForm
+                toolbar={
+                    <Toolbar>
+                        <SaveButton />
+                        <Button
+                            onClick={() => editorRef.current.commands.setContent('<h3>Here is my template</h3>')}
+                        >
+                            Use template
+                        </Button>
+                    </Toolbar>
+                }
+            >
+                <TextInput source="title" />
+                <RichTextInput 
+                    source="body" 
+                    editorOptions={{
+                        ...DefaultEditorOptions,
+                        onCreate: ({ editor }: { editor: Editor }) => {
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏Typescript complaints here that I haven't been able to resolve